### PR TITLE
Convert history page to html+css

### PR DESCRIPTION
**Pull request by mrdrogdrog**
This PR converts the history page from svg to html+css+svg+magic :sparkles:

image

<hr/>

image

### CODE REVIEW

**Comment by mrdrogdrog** [comment]
image

**Comment by DerMolly** [comment]
Please rebase and add reuse information
`reuse addheader --copyright="The HedgeDoc developers (see AUTHORS file)" --license="CC0-1.0" --explicit-license` for the .license files
`reuse addheader --copyright="The HedgeDoc developers (see AUTHORS file)" --license="CC0-1.0" --style="html"` for html / xml files

File: content/english/history.html [hunk 1]
```diff
@@ -8,12 +8,12 @@ <h3 class="mb-5">HedgeDoc is the community-driven fork of CodiMD, which is the o
 
 {{< history-entry graph="single point">}}
     <h3><time>May '15</time></h3>
-    The first commit in the history of HackMD is made.
+    <a rel="noopener noreferer" target="_blank" href="https://github.com/codimd/server/tree/61eb11d2">The first commit in the history of HackMD is made.</a>
```
Comment (ErikMichelson) [inline content/english/history.html:11]:
```suggestion
    <a rel="noopener noreferer" target="_blank" href="https://github.com/hedgedoc/hedgedoc/tree/61eb11d2">The first commit in the history of HackMD is made.</a>
```

**Review by ErikMichelson — CHANGES_REQUESTED** [review]

File: content/english/history.html (hunk 1 above)
Comment (mrdrogdrog) [reply content/english/history.html:11]: done

**Review by ErikMichelson — APPROVED** [review]

**Review by DerMolly — APPROVED** [review]

**Pull request merged**